### PR TITLE
Engineering#245 - Migrate from Travis to GH Actions

### PR DESCRIPTION
**Pull request by Samkeer1**
## Description
https://github.com/revelrylabs/engineering/issues/245

I'm still seeing references to Travis here, so we may need to do more than removing the Travis config here. I'm seeing our in-Travis repo settings has "build pushed branches" turned on, and we may need to adjust that:
https://app.travis-ci.com/github/revelrylabs/slax/settings

We should do this as soon as Travis is removed from the project.

https://docs.travis-ci.com/user/web-ui/#build-pushed-branches

### CODE REVIEW

**Review by jessrichmond — COMMENTED** [review]
can [bin/docker_pub](https://github.com/revelrylabs/slax/blob/f6f8ba47c0bc630aab14fc47e08d451794bf08ac/bin/docker_pub#L11) be updated to remove the travis reference?

**Review by jessrichmond — COMMENTED** [review]
~can [bin/docker_pub](https://github.com/revelrylabs/slax/blob/f6f8ba47c0bc630aab14fc47e08d451794bf08ac/bin/docker_pub#L11) be updated to remove the travis reference?~

sry, internet went in and out as i went to send this

**Comment by quinnmccourt** [comment]
we should remove coveralls as well

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+          - 5432:5432
+        options: >-
+          --health-cmd pg_isready --health-interval 10s
+          --health-timeout 5s --health-retries 5
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '3.3'
```
Comment (thebriz24) [inline .github/workflows/test.yml:33]: Do you need ruby in this?

Comment (Samkeer1) [reply .github/workflows/test.yml:33]: I did at this point, but not anymore since we're removing coveralls.

**Review by texastoast — APPROVED** [review]
gtm, watched it happen

**Pull request merged**